### DISCUSSION TITLE
What is difference between this component and reclyerlistview?

### DISCUSSION THREAD

**Discussion by jurica-penjgusic** [Shopify/flash-list#536] [Q&A]
I was wondering how this component is different from reclyerlistview from flipkart?
As I saw same folks are mainteners of both components and props of components are nearly identical?
If anybody knows the difference please share your knowledge 🙏

**Comment by leotm** [comment]
have you seen
- [FlashList uses recyclerlistview](https://github.com/Shopify/flash-list/blob/main/documentation/docs/known-issues.md#1-flashlists-rendered-size-is-not-usable-warning)
- https://github.com/Shopify/flash-list/discussions/501(#501)
- https://github.com/Shopify/flash-list/discussions/103
